File: SCC0600/exercíciosPy/ex25.py
l1 = int(input())
l2 = int(input())
l3 = int(input())

if l1 + l2 < l3 or l2 + l3 < l1 or l3 + l1 < l2:
  print('INVALIDO')
elif l1 == l2 and l2 == l3:
  print('EQUILATERO')
elif l1 == l2 or l1 == l3 or l2 == l3:
  print('ISOSCELES')
else:
  print('ESCALENO')

File: SCC0600/exerciciosC/ex25.c
#include <stdio.h>

int main (void) {
    int l1; int l2; int l3; char *tipo = "ESCALENO";
    scanf("%d", &l1);
    scanf("%d", &l2);
    scanf("%d", &l3);

    if (l1 + l2 < l3 || l1 + l3 < l2 || l2 + l3 < l1) {
        tipo = "INVALIDO";
    } else if (l1 == l2 && l2 == l3) {
        tipo = "EQUILATERO";
    } else if (l1 == l2 || l2 == l3 || l1 == l3) {
        tipo = "ISOSCELES";
    }

    printf("%s\n", tipo);
}